lib/membrane/video_compositor/queue/strategies/offline/element.ex

```diff
@@ -9,13 +9,16 @@ defmodule Membrane.VideoCompositor.Queue.Offline.Element do
 
   alias Membrane.{Pad, RawVideo, Time}
   alias Membrane.VideoCompositor
-  alias Membrane.VideoCompositor.{CompositorCoreFormat, Queue}
-  alias Membrane.VideoCompositor.Queue.Offline.State, as: OfflineState
+  alias Membrane.VideoCompositor.CompositorCoreFormat
+  alias Membrane.VideoCompositor.Queue.Contracts
   alias Membrane.VideoCompositor.Queue.State
   alias Membrane.VideoCompositor.Queue.State.{HandlerState, PadState}
+  alias Membrane.VideoCompositor.Queue.Strategies.Offline.State, as: OfflineState
 
   def_options vc_init_options: [
-                spec: VideoCompositor.init_options()
+                spec: VideoCompositor.init_options(),
+                description: "Init options of VideoCompositor.
+                Used in `c:Membrane.VideoCompositor.Handler.handle_init/1` callback"
```

Comment:
How about something similar to that:
```suggestion
                description: "Options to initialize VideoCompositor.
                Passed in `c:Membrane.VideoCompositor.Handler.handle_init/1` callback invocation."
```
?